ValidationMixin.js

```diff
@@ -44,18 +43,40 @@ var ValidationMixin = {
    * Convenience method to validate a key via an event handler. Useful for
    * onBlur, onClick, onChange, etc...
    *
-   * @param {?String} State key to validate
+   * Usage: <input onBlur={this.handleUnfocusFor('password')} .../>
+   *
+   * @param {?String} key to validate
    * default is false.
    * @return {function} validation event handler
    */
-  handleValidation: function(key) {
-    return function(event) {
+  handleUnfocusFor: function(key) {
```

Comment:
`handleUnfocusFor` does not account for the fact this event handler could be attached to an `onChange` event or really any event. I think we need to stick with the generic `handleValidation` or something like it to emphasize this fact.